main.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"encoding/csv"
+	"flag"
+	fmt "fmt"
+	_ "go/parser"
+	"io"
+	"os"
+)
+
+//
+type QAPair struct {
+	Quest string
+	Ans   string
+}
+
+func readProblems() [][]string {
+	//TO DO:read in a quiz provided via a CSV file, GO CSV file plug in
+	//reading in the CSV file
+	csvFlag := flag.String("csvFile", "problems.csv", "CSV file question,answer")
+	flag.Parse()
+
+	csvFile, err := os.Open("problems.csv")
```

Comment:
I think using the flag here instead of hardcoding the filename might streamline future updates or reusability of this code
```suggestion
	csvFile, err := os.Open(*csvFlag)
```

Reply:
Nice catch! This will enable other files to be used with the program.